install-fabtests.sh

```diff
@@ -31,8 +31,8 @@ else
 fi
 # .bashrc and .bash_profile are loaded differently depending on distro and
 # whether the shell is interactive or not, just do both to be safe.
-echo "export LD_LIBRARY_PATH=${LIBFABRIC_INSTALL_PATH}/lib/:\$LD_LIBRARY_PATH" >> ~/.bash_profile
-echo "export LD_LIBRARY_PATH=${LIBFABRIC_INSTALL_PATH}/lib/:\$LD_LIBRARY_PATH" >> ~/.bashrc
+#echo "export LD_LIBRARY_PATH=${LIBFABRIC_INSTALL_PATH}/lib/:\$LD_LIBRARY_PATH" >> ~/.bash_profile
```

Comment:
I thought this is needed for fabtests to compile?

Reply:
It is not need for installer canary test because installer will update ld.so.conf.d, but it might be needed for libfabric CI runs. I commented them out because I though they will prevent Intel MPI U6 from using its internal libfabric, and I just realized it would not happen because we source mpivars.sh which would put Intel MPI's internal libfabric at the beginning of LD_LIBRARY_PATH. I will push a new PR.